package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "yapople",
```

Comment:
Just remove `package.json` from commit, it breaks package name and version.
Or, if you want to mention you as a contributor, you can do it like this: https://nodejs.dev/learn/the-package-json-guide#contributors
And don't forget about these gyus :smile: https://github.com/agsh/yapople/graphs/contributors

Reply:
I modified package.json confirm.